citools/dump_failures.py

```diff
@@ -13,14 +13,15 @@
 from collections import defaultdict
 from typing import Dict, List
 
-from mozci.push import Push
+from mozci.push import Push,PushStatus, Regressions
```

Comment:
nit: leave a ' ' after the ','.

it would look like: `...import Push, PushStatus, ...`